test-projects/pizza-ordering/cli/order/dynamic-sync-accepts/dynamic-sync-accepts.spec.js

```diff
@@ -0,0 +1,23 @@
+function dataAllows() {
+	return ['a', 'b', 'c', 'd'];
+}
+
+function testAllows() {
+	return ['a1', 'b2', 'c3', 'd4'];
```

Comment:
Same as the other file:

```suggestion
	return ['a1', 'b1', 'c1', 'd1'];
}

function testAllows() {
	return ['a2', 'b2', 'c2', 'd2'];
```